content/docs/intro/cloud-providers/digitalocean/setup.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Setup
+
+aliases: ["/docs/reference/clouds/digitalocean/setup/"]
+---
+
+The [Pulumi DigitalOcean provider]({{< relref "./" >}}) uses the DigitalOcean SDK to manage and provision resources.
```

Comment:
Maybe insignificant but sometimes we say "X Pulumi Provider" or "Pulumi X Provider" when referring to different providers  (not on this page but on others) - I'm not sure which we want to be consistent with though.

Reply:
Pick one and be consistent